components/Paybutton/PaybuttonTrigger.tsx

```diff
@@ -133,23 +187,45 @@ export default ({ paybuttonId }: IProps): JSX.Element => {
                 {/* Only triggers if payment &gt; X */}
               </div>
               <div className={style.btn_row2}>
-                {(error === undefined || error === '') ? null : <div className={style.error_message}>{error}</div>}
-                {(successText === undefined || successText === '') ? null : <div className={style.success_message}>{successText}</div>}
+                {(posterError === undefined || posterError === '') ? null : <div className={style.error_message}>{posterError}</div>}
+                {(posterSuccessText === undefined || posterSuccessText === '') ? null : <div className={style.success_message}>{posterSuccessText}</div>}
                 <div>
-                  <button disabled={disableSubmit} type='submit' className='button_main'>{currentTriggerId === undefined ? 'Create' : 'Update'}</button>
+                  <button disabled={disablePosterSubmit} type='submit' className='button_main'>{currentPosterTriggerId === undefined ? 'Create' : 'Update'}</button>
                 </div>
                 <div>
                   {
-                    currentTriggerId !== undefined &&
-                      <button type='button' onClick={() => setClearModal(true)} className={style.delete_btn}>Clear</button>
+                    currentPosterTriggerId !== undefined &&
+                      <button type='button' onClick={() => setClearModal('poster')} className={style.delete_btn}>Clear</button>
+                  }
+                </div>
+              </div>
+            </div>
+            </form>
+          <form onSubmit={(e) => { void handleSubmitEmailTrigger(getSubmitTriggerHandler('email'))(e) }} method='post'>
+            <h5>Receive Email</h5>
+            <div>
+              <label htmlFor="emails">Receive email</label>(<span>{emailCredits} credits</span>)
```

Comment:
Change 'Receive Email' to 'Send email' to match post URL section.

Reply:
done

Reply:
Can you match the capitalization (Send request vs Send Email)?

Reply:
Also change 'Receive email' below it to just 'Email:'.

Reply:
Also while you're in there, remove the ':' from 'URL:'.

Reply:
Addressed, also modified the placeholder to be a little more clear.